quiz.js

```diff
@@ -0,0 +1,64 @@
+"use strict"
+
+	CarLot = (function(oldCarLot){
+
+    oldCarLot.displayCars = function(inventory){
+        var inCars = document.getElementsByClassName("row")[0];
```

Comment:
What happens here if you add another car? I think this variable is a little too specific. You may want to handle it in the `for` loop (`inCars[i]`).